static/templates/disbursements.hbs

```diff
@@ -45,6 +49,20 @@
 </div>
 
 <div class="panel__row">
+  <h4 class="panel__title">Filing information</h4>
+  <table>
+    <tr>
+      <td class="panel__term">Filing date</td>
+      <td>{{{ datetime receipt_date }}}</td>
+    </tr>
+    <tr>
+      <td class="panel__term">View report page</td>
+      <td><a href="{{{ pdf_url }}}" target="_blank">View on fec.gov</a></td>
```

Comment:
We are showing the page that the record is on, which is handy because the reports can be thousands of pages.